tools/pubsys/src/kit/publish_kit/mod.rs

```diff
@@ -26,52 +25,36 @@ pub(crate) struct PublishKitArgs {
     /// The build id of the kit that should be published
     #[arg(long)]
     build_id: String,
-}
 
-macro_rules! docker {
-    ($arg: expr) => {{
-        let result = Command::new("docker")
-            .args($arg)
-            .output()
-            .await
-            .context(error::CommandSnafu { command: "docker" })?;
-        ensure!(
-            result.status.success(),
-            error::PublishFailSnafu {
-                reason: String::from_utf8_lossy(&result.stderr)
-            }
-        );
-        result.stdout
-    }};
+    /// The image tool to use when pushing the kit. Valid arguments are docker and crane.
+    #[arg(long)]
+    image_tool: Option<String>,
```

Comment:
Does this get used? Seems like we just call the `image_tool` helper function.

Reply:
whoops, this was a remnant from my initial implementation of `crane` support before #305. did not do a good job of cleaning up this file 🤦